### ISSUE TITLE
MSFR loss

### ISSUE THREAD

**Issue by misakshoyan** [INVOKERer/DeepRFT#13]
Hi @INVOKERer.
In MIMO-UNET, where pytorch version is 1.4, the FFT for MSFR loss is calculated as
`torch.rfft(image, signal_ndim=2, normalized=False, onesided=False)`

Could you, please, tell me how do you port it to torch.fft.rfft in pytorch 1.8?

**Comment by INVOKERer** [comment]
You can use torch.fft.rfft2/fft2/fftn/rfftn to calculate MSFR. We use torch.fft.fft2 to  calculate MSFR.

**Comment by misakshoyan** [comment]
torch.fft.fft2 or torch.fft.rfft2?
And, could you, please, share your one line code with me if it is possible?

**Comment by INVOKERer** [comment]
torch.mean(abs(torch.fft.fft2(x) - torch.fft.fft2(y)))

**Comment by misakshoyan** [comment]
Thank You!

**Comment by misakshoyan** [comment]
I get the following warning message after loss.backward():
"Warning: Casting complex values to real discards the imaginary part (function operator())"

Is this expected?

When I use rfft2,
"torch.mean(abs(torch.fft.rfft2(x) - torch.fft.rfft2(y)))"
warning disappears.
